ingestion/src/metadata/utils/helpers.py

```diff
@@ -270,3 +271,12 @@ def find_in_list(element: Any, container: Iterable[Any]) -> Optional[Any]:
     :return: element or None
     """
     return next(iter([elem for elem in container if elem == element]), None)
+
+
+def find_column_in_table(column_name: str, table: Table) -> Optional[Column]:
+    """
+    If the column exists in the table, return it
+    """
+    return next(
+        iter([col for col in table.columns if col.name.__root__ == column_name]), None
+    )
```

Comment:
For these expression you can simply use a generator expression

```
return next((col for col in table.columns if col.name.__root__ == column_name), None)
```

Reply:
such a good point here. I wrote this line down a few months ago and never looked back, thanks for the improvement, it is great